tests/fixtures/blockchainApplicationsManage.js

```diff
@@ -3,7 +3,7 @@ const blockchainApplicationsManage = [
     name: 'Lisk',
     chainID: 'aq02qkbb35u4jdq8szo3pnsq',
     state: 'active',
-    apis: [{ rest: 'https://service.lisk.com', rpc: 'wss://service.lisk.com' }],
+    apis: [{ rest: 'http://104.248.241.229:9901', rpc: 'wss://104.248.241.229:9901' }],
```

Comment:
I think we should not use this? rather we need to setup the local server and access them as the `104.248.241.229` is only available for temporary usage.

Reply:
in local we using mocks! by the way we will update it later and now we need it like this